app/src/main/java/org/coepi/android/ui/onboarding/binders/LargeOnboardingCardBinder.kt

```diff
@@ -0,0 +1,35 @@
+package org.coepi.android.ui.onboarding.binders
+
+import android.view.View
+import io.reactivex.subjects.Subject
+import kotlinx.android.synthetic.main.card_onboarding_large.view.*
+import org.coepi.android.ui.onboarding.OnboardingCard
+import org.coepi.android.ui.onboarding.OnboardingCard.LargeOnboardingCard
+import org.coepi.android.ui.onboarding.OnboardingClickEvent
+
+object LargeOnboardingCardBinder: OnboardingCardBinder {
+
+    override fun bind(
+        itemView: View,
+        onboardingCard: OnboardingCard,
+        onboardingEventSubject: Subject<OnboardingClickEvent>
+    ) {
+        onboardingCard as LargeOnboardingCard
```

Comment:
Let's put the force-casts in the adapter, so they're all together and it's consistent with the rest of the code. Pass `LargeOnboardingCard` to binder (also in the others)